code_samples/sql_json.js

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) 2008-2021, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+const { Client, SqlColumnType, HazelcastJsonValue } = require('hazelcast-client');
+
+(async () => {
+    const client = await Client.newHazelcastClient();
+    const mapName = 'jsonMap' + Math.floor(Math.random() * 10000);
+
+    // In order to use the map in SQL a mapping should be created.
+    const createMappingQuery = `
+            CREATE OR REPLACE MAPPING ${mapName}  (
+                __key BIGINT,
+                this JSON
+            )
+            TYPE IMAP
+            OPTIONS (
+                'keyFormat' = 'bigint',
+                'valueFormat' = 'json'
+            )
+        `;
+    await client.getSql().execute(createMappingQuery);
+
+    await client.getSql().execute(`INSERT INTO ${mapName} VALUES (1, ?)`,
+        [new HazelcastJsonValue(JSON.stringify({age: 1}))]
```

Comment:
Same as documentation, it would be far easier to use JS objects here for the user

Reply:
ok changed with a note